docs/containers/local-process-kubernetes-vs-code.md

```diff
@@ -0,0 +1,193 @@
+
+> **Important**: You must have *Owner* or *Contributor* access to your cluster in order to run the script.
+
+```azurecli-interactive
+git clone https://github.com/Azure/dev-spaces
+cd dev-spaces/
+chmod +x ./local-process-quickstart.sh
+./local-process-quickstart.sh -g MyResourceGroup -n MyAKS
+```
+
+Navigate to the sample application running your cluster by opening its public URL, which is displayed in the output of the installation script.
+
+```console
+$ ./local-process-quickstart.sh -g MyResourceGroup -n MyAKS
+Defaulting Dev spaces repository root to current directory : ~/dev-spaces
+Setting the Kube context
+...
+To try out the app, open the url:
+dev.bikesharingweb.EXTERNAL_IP.nip.io
+```
+
+In the above sample, the public URL is `dev.bikesharingweb.EXTERNAL_IP.nip.io`.
+
+## Connect to your cluster and debug a service
+
+On your development computer, download and configure the Kubernetes CLI to connect to your Kubernetes cluster using [az aks get-credentials][az-aks-get-credentials].
+
+```azurecli
+az aks get-credentials --resource-group MyResourceGroup --name MyAKS
+```
+
+Open *dev-spaces/samples/BikeSharingApp/Bikes* from the [Azure Dev Spaces Bike Sharing sample application][bike-sharing-github] in Visual Studio Code. Open the Azure Kubernetes Service extension and select the *dev* namespace in the *MyAKS* cluster.
+
+![Select Namespace](images/local-process-kubernetes-vs-code/select-namespace.png)
+
+Use the `npm install` command to install the dependencies for the application.
+
+```console
+npm install
+```
+
+Select the *Debug* icon on the left and select *Local Process with Kubernetes (Preview)* at the top.
+
+![Choose Local Process with Kubernetes](images/local-process-kubernetes-vs-code/choose-local-process.png)
+
+Click on the start button next to *Local Process with Kubernetes (Preview)*. The first time you run this launch configuration you are prompted to configure the service you want to replace, the port to forward from your development computer, and the launch task to use.
+
+Choose the *bikes* service.
+
+![Choose Service](images/local-process-kubernetes-vs-code/choose-service.png)
+
+All traffic in the Kubernetes cluster is redirected for the *bikes* service to the version of your application running in your development computer. Local Process with Kubernetes also routes all outbound traffic from the application back to your Kubernetes cluster.
+
+> **Important**: You can only redirect services that have a single pod.
+
+After you select your service, you are prompted to enter the TCP port for your local application. For this example, enter *3000*.
+
+![Connect choose port](images/local-process-kubernetes-vs-code/choose-port.png)
+
+Choose *Launch via NPM* as the launch task.
+
+![Connect choose launch task](images/local-process-kubernetes-vs-code/choose-launch.png)
+
+> **Note**: You will be prompted to allow the *KubernetesDNSManager* to run elevated and modify your hosts file.
```

Comment:
Hi @ghogen , wanted to let you know that KubernetesDNSManager will be renamed EndpointManager by 7/14.  Thanks!

Reply:
@amsoedal @greenie-msft Should we wait until 7/14 before publishing the name change?

Reply:
Hi @ghogen, the rename should go into our next release, so it would be great to have the name published in sync with that release.  @greenie-msft I don't recall if our next release is still 7/14?